entity-framework/core/what-is-new/ef-core-8.0/whatsnew.md

```diff
@@ -1127,3 +1127,468 @@ In addition to the enhancements described above, EF8 Preview 2 also includes som
 
 - [Configuration to opt out of occasionally problematic SaveChanges optimizations](https://github.com/dotnet/efcore/issues/29916)
 - [Add convention types for triggers](https://github.com/dotnet/efcore/issues/28687)
+
+## New in EF8 Preview 4
```

Comment:
Should we just have a flat list, do we think it matters which features were introduced in which preview?

Reply:
Not sure. We should discuss.